jni/nnstreamer.mk

```diff
@@ -1,8 +1,11 @@
 # This mk file defines common features to build NNStreamer library for Android.
 
 ifndef NNSTREAMER_ROOT
-$(warning NNSTREAMER_ROOT is not defined! Using $(LOCAL_PATH)/.. )
-NNSTREAMER_ROOT := $(LOCAL_PATH)/..
+$(error NNSTREAMER_ROOT is not defined!)
+endif
```

Comment:
Can you define ```NNSTREAMER_ROOT``` and its friends in nnstreamer.mk and remove them from other .mk files?

Reply:
NNSTREAMER_ROOT is defined in gradle build script (system.env or local properties of android-studio project)